static/js/publisher/release/components/releasesTable/releaseCell.js

```diff
@@ -93,11 +95,16 @@ const ReleasesTableReleaseCell = props => {
   };
 
   function handleHistoryIconClick(arch, risk, track, branchName) {
+    triggerGAEvent("click-history", `${track}/${risk}/${branchName}/${arch}`);
     toggleHistoryPanel({ arch, risk, track, branch: branchName });
   }
 
   function undoClick(revision, channel, event) {
     event.stopPropagation();
+    triggerGAEvent(
```

Comment:
This could also be dispatched in the pendingReleases action

Reply:
Will keep this where it is so we have both `click-revert` and `click-save` defined in the component